hw03_vesselsj2.py

```diff
@@ -0,0 +1,152 @@
+#################################################################################
+# Author: Jaren Vessels
+# Username: vesselsj2
+#
+# Assignment: T03: Boustrophedon Turtles
+# Purpose: To draw an illusion with a person floating on it
+#################################################################################
+# Acknowledgements:
+#   Runestone Academy Chapter 5: Modules
+#   Used AI to help debug random module that was not letting me randomize the color of my screen and turtles
+#   Used AI to help me debug my illusion pattern that was not increasing in size on each repetition
+#################################################################################
+
+import turtle
+import random
+
+
+def ran_col():
+    """
+    Creates the random rgb generator for my code
+    """
+    r = int(random.random() * 255)
+    g = int(random.random() * 255)
+    b = int(random.random() * 255)
+    return r, g, b
+
+
+def draw_person(ben, wn):
+    wn.bgcolor(ran_col())
+    """
+    Randomizes the color of the background screen when this function is called
+    """
+    ben.speed(0)
+    ben.pensize(10)
+    ben.penup()
+    ben.goto(-100, 50)
+    """
+    Setting up the position of ben to draw the person
+    """
```

Comment:
Comments typically are `# comment text...`, not triple-quoted strings. There are exceptions to this rule (e.g., very lengthy comments in very complex codes), but this is not one of those cases.

Reply:
Multiple places to fix this.